config/richrenderer.ini

```diff
@@ -0,0 +1,7 @@
+[styles]
+repr.call = bold italic #00d8eb
+repr.ipv6 = yellow 
+repr.number = #008df8
+repr.str = bright_blue
+table.header = bold italic #00d8eb
+table.border = #e1251b
```

Comment:
An empty new line is required for the EOF point.

Reply:
Added